src/chart/Bubble.js

```diff
@@ -35,6 +36,23 @@
         return retVal;
     };
 
+    Bubble.prototype.enter = function () {
+        if (this.useTooltip()) {
```

Comment:
I don't think we want this test here - otherwise you can't disable/enable it after it has rendered once.